Story/Extinction.cs

```diff
@@ -0,0 +1,58 @@
+//cs_include Scripts/CoreBots.cs
+//cs_include Scripts/CoreStory.cs
+using Skua.Core.Interfaces;
+
+public class Extinction
+{
+    public IScriptInterface Bot => IScriptInterface.Instance;
+    public CoreBots Core => CoreBots.Instance;
+    public CoreStory Story = new CoreStory();
+
+    public void ScriptMain(IScriptInterface Bot)
+    {
+        Core.SetOptions();
+
+        StoryLine();
+
+        Core.SetOptions(false);
+    }
+
+    public void StoryLine()
+    {
+        if (Core.isCompletedBefore(3864))
+            return;
+
+        Story.PreLoad(this);
+
+    //Chopped HAMM 3855
+        Story.MapItemQuest(3855, "extinction", 2956, 5);
```

Comment:
spacing here is fine but it's better to align them with thier respective //{questName} {questId} 
if you are not using pretyy formatter extentions on vs code use it and ctrl+ alt + f